stumpy/floss.py

```diff
@@ -139,6 +155,9 @@ def _cac(I, L, bidirectional=True, excl_factor=5, custom_iac=None, seed=0):
         A custom idealized arc curve (IAC) that will used for correcting the
         arc curve
 
+    seed : int
+        TODO
```

Comment:
I wasn't sure on what might be a good description for this.
But I added it just for you :)

Reply:
Great observation! I can add the description afterward